src/Uno.UI.Runtime.Skia/SkiaHost.cs

```diff
@@ -5,14 +5,14 @@ public abstract class SkiaHost
 {
 	internal Action? AfterInit { get; set; }
 
-	public void Run()
+	public async Task Run()
 	{
 		Initialize();
 		AfterInit?.Invoke();
-		RunLoop();
+		await RunLoop();
 	}
```

Comment:
This would be breaking. I think we should add a RunAsync method instead.

Reply:
@jeromelaban I thought existing code could still be calling `Run` without breaking?

Reply:
It would break, since Run is blocking, and calling it without await will go though without blocking.